liquigraph-examples/dagger2/src/test/java/org/liquigraph/examples/dagger2/ApplicationTest.java

```diff
@@ -17,13 +17,13 @@
 
 import java.io.IOException;
 import java.net.ServerSocket;
+import org.junit.BeforeClass;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.junit.BeforeClass'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)